curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641daabed8d0584b1150c953.md

```diff
@@ -28,22 +28,21 @@ assert.match(code, /<p\s*class=("|')bio\1>/);
 You should check if the length of the `bio` text is greater than `50`. 
 
 ```js
-assert.match(code, /<p\s*class=("|')bio\1>\$\{\s*bio\.length\s*>\s*50/)
+assert.match(code, /<p\s*class=("|')bio\1>\s*\$\{\s*bio\.length\s*>\s*50/)
 ```
 
 If the `bio` text is greater than `50` characters, you should extract the first 50 characters with `slice()` and replace the rest with `...`. Don't forget that indexes are zero-based.
 
 ```js
-assert.match(code, /<p\s*class=("|')bio\1>\$\{\s*bio\.length\s*>\s*50\s*\?\s*bio\.slice\(\s*0\,\s*50\s*\)\s*\+\s*("|')\.\.\.\2\s*\:/)
+assert.match(code, /<p\s*class=("|')bio\1>\s*\$\{\s*bio\.length\s*>\s*50\s*\?\s*bio\.slice\(\s*0\,\s*50\s*\)\s*\+\s*("|')\.\.\.\2\s*\:|<p\s*class="bio">\s*\$\{bio\.length\s*>\s*50\s*\?\s*`\$\{bio\.slice\(0,\s*50\)\}\.\.\.`\s*:\s*bio\}<\/p>/);
```

Comment:
Looks like something might got lost in the meantime. Committed changes from last suggestions were reverted?

Reply:
I dont understand..

Reply:
Changes from https://github.com/freeCodeCamp/freeCodeCamp/pull/53178#pullrequestreview-1838559849 appear to be lost.

Reply:
I look into it.

Reply:
```suggestion
assert.match(code, /<p\s*class\s*=\s*("|')bio\1\s*>\s*\$\{\s*bio\.length\s*>\s*50\s*\?\s*(?:bio\.slice\(\s*0\s*\,\s*50\s*\)\s*\+\s*("|')\.\.\.\2|`\$\{\s*bio\.slice\(\s*0\s*,\s*50\s*\)\s*\}\.\.\.`)\s*:/);
```